recipes/lhorizon/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lhorizon" %}
+{% set version = "1.0.1a2" %}
```

Comment:
Do you plan to update this to a release before we merge?

Reply:
sure thing! I've done a little cleaning and made a 1.1.0 release.